component-samples/demo/README.MD

```diff
@@ -0,0 +1,233 @@
+
+3. Complete Ownership Voucher Extension by using the API `GET /api/v1/mfg/vouchers/<serial_no>` and save the Ownership Voucher. Please refer to [PRI Manufacturer REST APIs](manufacturer/README.md/#fdo-pri-manufacturer-rest-apis) for more information about the API.
+
+4. Start the PRI RV Sample as per the steps outlined in [RV README](rv/README.md).
+
+5. Start the PRI Owner Sample as per the steps outlined in [Owner README](owner/README.md). Import the extended Ownership Voucher from Step#3 into the Owner database by using the API `POST /api/v1/owner/vouchers/`. Please refer to [FDO PRI Owner REST APIs](owner/README.md/#fdo-pri-owner-rest-apis) for more information about the API. Optionally, if ServiceInfo transfer is needed, please refer to [Enabling ServiceInfo transfer](#enabling-serviceinfo-transfer).
+
+6. Complete Transfer Ownership 1 and 2 (TO1 and TO2) by starting the FDO PRI HTTP Java Device Sample again. The previously created 'credentials.bin' from Step#2 will be used directly by the Device.
+
+***NOTE***: Credential Reuse protocol is enabled by default, that is, after a successful onboarding the device credentials remain unchanged. To enable the Resale protocol instead, wherein, after a successful onboarding the device credentials are replaced, update the fields `REPLACEMENT_GUID` and/or `REPLACEMENT_RVINFO` in Owner `TO2_DEVICES` table by using the API `POST /api/v1/owner/setupinfo?id=<device_guid>` before starting TO2. Please refer to [FDO PRI Owner REST APIs](owner/README.md/#fdo-pri-owner-rest-apis) for more information about the API.
+
+# ServiceInfo setup between FDO PRI HTTP Java Device Sample and FDO PRI Owner Sample
+
+The FDO PRI HTTP Java Device Sample currently supports `fdo_sys` and `fidoalliance` module for interpreting received owner ServiceInfo and `devmod` module to share device ServiceInfo with Owner.
+
+- `fdo_sys` Owner ServiceInfo module: This module supports the following 3 message names as listed below to interpret the ServiceInfo as received from the Owner. The basic functionality of this module is to support payload/script transfers and basic command execution.  A sample format looks like 'fdo_sys:filedesc=filename, fdo_sys:write=filecontent,fdo_sys:exec=command-to-execute'.
+
+    *filedesc* - The name to be given to the file once it is transferred. Upon receiving this, device creates a file with the given name and opens stream to write into it.
+
+    *write* - The payload/content (script, binaries, and others) that is sent to the device. Upon receiving this, device writes the content into the open stream as given by the preceding 'filedesc' message.
+
+    *exec* - The command that will be executed at the device. Device executes the command as received.
+
+    *exec-cb* - This command returns the status / progress of the process running.
```

Comment:
Do we need to give any examples?

Reply:
We can add a sample SVI instruction using all commands